### PR TITLE
Added links to Wiki and Setup guide to the config

### PR DESCRIPTION
**Pull request by onebeastchris**
Hey!
Personally, i think it could be helpful if the configuration file would include a link to both the wiki as a whole. Additionally, some guidance pointing to the setup guide specifically sounds beneficial.

Not set in stone about this one, feel free to close if deemed unnecessary.

### CODE REVIEW

File: core/src/main/resources/config.yml [hunk 1]
```diff
@@ -5,6 +5,11 @@
 #
 # GitHub: https://github.com/GeyserMC/Geyser
 # Discord: https://discord.geysermc.org/
+# Wiki: https://wiki.geysermc.org/
+#
+# DISCLAIMER: See https://wiki.geysermc.org/geyser/setup/ for the setup guide. Many video tutorials are outdated.
+# In most cases, espescially with server hosting providers, further hosting specific configuration is required.
```
Comment (Camotoy) [inline core/src/main/resources/config.yml:11]: Especially

File: core/src/main/resources/config.yml [hunk 2]
```diff
@@ -5,6 +5,11 @@
 #
 # GitHub: https://github.com/GeyserMC/Geyser
 # Discord: https://discord.geysermc.org/
+# Wiki: https://wiki.geysermc.org/
+#
+# DISCLAIMER: See https://wiki.geysermc.org/geyser/setup/ for the setup guide. Many video tutorials are outdated.
```
Comment (Camotoy) [inline core/src/main/resources/config.yml:10]: DISCLAIMER isn't the right tone, I think. Maybe NOTICE instead?

File: core/src/main/resources/config.yml (hunk 1 above)
Comment (onebeastchris) [reply core/src/main/resources/config.yml:11]: oh yea, right
one of the words i find - especially - hard to spell

File: core/src/main/resources/config.yml (hunk 2 above)
Comment (onebeastchris) [reply core/src/main/resources/config.yml:10]: fits better, yep
caps are okay, right?

**Review by Konicai — APPROVED** [review]

**Comment by matsyyy** [comment]
check failed xddd
how come this tiny change takes this long? i dont need it, but others might

**Comment by Konicai** [comment]
Reopen under a new branch

**Pull request closed without merging**